### ISSUE TITLE
Cannot converge in the first epoch.

### ISSUE THREAD

**Issue by synsin0** [deku-0621/TEST-TIME-FEW-SHOT-OBJECT-DETECTION-IN-THE-WILD#1]
I train the model without backbone with a batch_size of 32 in one GPU. The loss reduces to 20.x as a half but improves to 60-70 afterwards. May you give some instructions on why the loss cannot converge?

**Comment by deku-0621** [comment]
Because of the original conflict of interest, now the company does not allow the latest code. Backbone and data processing can be fine-tuned. Or you can send an email to Yuan Li to ask if he can provide the latest version of the code.

**Comment by deku-0621** [comment]
his e-mail address, lyn@intelliclound.ai
